File: Restaurant/Restaurant.Web/Modules/ServerTypes/Default/BrandForm.ts
import { StringEditor, PrefixedContext, initFormType } from "@serenity-is/corelib";

export interface BrandForm {
    BrandName: StringEditor;
    Photo: StringEditor;
    PhoneNember: StringEditor;
    WhtasApp: StringEditor;
    FaceBook: StringEditor;
    Twitter: StringEditor;
    Instagram: StringEditor;
    Tiktok: StringEditor;
    Telegram: StringEditor;
    Description: StringEditor;
}

export class BrandForm extends PrefixedContext {
    static readonly formKey = 'Default.Brand';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!BrandForm.init)  {
            BrandForm.init = true;

            var w0 = StringEditor;

            initFormType(BrandForm, [
                'BrandName', w0,
                'Photo', w0,
                'PhoneNember', w0,
                'WhtasApp', w0,
                'FaceBook', w0,
                'Twitter', w0,
                'Instagram', w0,
                'Tiktok', w0,
                'Telegram', w0,
                'Description', w0
            ]);
        }
    }
}

File: Restaurant/Restaurant.Web/Modules/Default/Brand/BrandForm.cs
using Serenity.ComponentModel;

namespace Restaurant.Default.Forms;

[FormScript("Default.Brand")]
[BasedOnRow(typeof(BrandRow), CheckNames = true)]
public class BrandForm
{
    public string BrandName { get; set; }
    public string Photo { get; set; }
    public string PhoneNember { get; set; }
    public string WhtasApp { get; set; }
    public string FaceBook { get; set; }
    public string Twitter { get; set; }
    public string Instagram { get; set; }
    public string Tiktok { get; set; }
    public string Telegram { get; set; }
    public string Description { get; set; }
}